go.mod

```diff
@@ -5,8 +5,8 @@ go 1.18
 require (
```

Comment:
Looks like linter is stuck on `replace` statement, but it has been removed and required functionality has been merged upstream 👍🏻